components/thumbnail/basic/src/index.js

```diff
@@ -1,15 +1,38 @@
 import React, {Component} from 'react'
 import PropTypes from 'prop-types'
+import ImagePlaceholder from '@schibstedspain/sui-image-placeholder'
 
 class ThumbnailBasic extends Component {
+  static BASE_CLASS = 'sui-ThumbnailBasic'
+  static IMAGE_CLASS = 'sui-ThumbnailBasic-image'
+  static CAPTION_CLASS = 'sui-ThumbnailBasic-caption'
+  static CAPTION_TEXT_CLASS = 'sui-ThumbnailBasic-caption-text'
+
+  get _imagePlaceholderProps () {
+    const props = Object.assign(
+      {},
+      this.props
+    )
+
+    delete props.captionText
+    return props
+  }
+
   render () {
-    const { image, captionText } = this.props
+    const {captionText} = this.props
     return (
-      <figure className='sui-ThumbnailBasic'>
-        <img className='sui-ThumbnailBasic-image' src={image.src} alt={image.alt} />
-        <figcaption className='sui-ThumbnailBasic-caption'>
-          <span>{captionText}</span>
-        </figcaption>
+      <figure className={ThumbnailBasic.BASE_CLASS}>
+        <ImagePlaceholder
```

Comment:
I'd say it can be done like the following:

```
<ImagePlaceholder {...this.props, captionText: undefined}   />
```

Up to you.

Reply:
Would like to use but our preset doesn't allow it!

![captura de pantalla 2017-10-24 a las 10 13 05](https://user-images.githubusercontent.com/17549662/31931591-f980bbca-b8a3-11e7-9623-1fdad5954a5e.png)